src/core/reducers/addons.js

```diff
@@ -293,7 +293,11 @@ export const getAddonBySlug = (
   state: AppState,
   slug: string,
 ): AddonType | null => {
-  const addonId = state.addons.bySlug[slug];
+  if (typeof slug !== 'string') {
```

Comment:
This doesn't seem to be covered by a test, at least not a test where `slug` is neither a string nor `null` nor `undefined`.

Reply:
Looking again, wouldn't Flow protect us from this ever happening?

Reply:
Flow won't be helpful at runtime.

There are 3 test cases for string, null or undefined. I think that's enough, but if yu like, I can still add another one with a number.

Reply:
I'm not sure it's about what I like. If the code is written to return a `null` when a number is passed, it seems like we should have a test for that. Otherwise it could have just been `if(!slug)`.

Reply:
fair enough, updated!